### PR TITLE
Release v2021.09.02.r52.2

### PR DESCRIPTION
**Pull request by shhdgit**

### CODE REVIEW

**Comment by breeswish** [comment]
In order to fix bugs exists in v4.0.x, I guess we need a new branch release-4.0.

**Comment by shhdgit** [comment]
RIght! And what about 5.0.x, 5.1.x?

**Comment by breeswish** [comment]
We can keep 4.0.x, 5.0.x and 5.1.x using the same TiDB Dashboard release.

File: release-version [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # This file specifies the TiDB Dashboard internal version, which will be printed in `--version`
 # and UI. In release branch, changing this file will result in publishing a new version and tag.
-2021.09.02.r52.1
+2021.11.25.r52.1
```
Comment (breeswish) [inline release-version:3]: As a hot patch I would prefer `2021.09.02.r52.2`

**Comment by shhdgit** [comment]
/cc @breeswish 
PTAL, thanks~

**Review by breeswish — APPROVED** [review]

**Comment by breeswish** [comment]
LGTM. Maybe you will also want to apply changes from https://github.com/pingcap/tidb-dashboard/commit/9d9aa849491b723e744076520b2d214ead878088 so that you don't need to create PD PR manually.

**Pull request merged**